src/services/config.js

```diff
@@ -30,5 +30,6 @@ if (admin_pass) {
 
 config.auth = {
   basic_enabled: Config.castBool(config.getEnv('AUTH_BASIC_ENABLED'), true),
-  http_signature_enabled: Config.castBool(config.getEnv('AUTH_HTTP_SIGNATURE_ENABLED'), true)
+  http_signature_enabled: Config.castBool(config.getEnv('AUTH_HTTP_SIGNATURE_ENABLED'), true),
+  client_certificates_enabled: Config.castBool(config.getEnv('CLIENT_CERTS_ENABLED'), true)
```

Comment:
This is a good point of discussion. To use client-cert auth, https must be enabled. If https is enabled, we need a way to configure tls, passing in the certificate options. This should probably exist in five-bells-shared, where we can have a common way to specify tls certificates and keys. I can make a separate PR to do that.